### PR TITLE
coap-gateway: update device status online in shortime than exp token

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

**Review by rafajpet — APPROVED** [review]

File: coap-gateway/service/config.go [hunk 1]
```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/plgd-dev/kit/sync/task/queue"
 )
 
+type DeviceStatusValidityConfig struct {
```
Comment (ondrejtomcik) [inline coap-gateway/service/config.go]: `DeviceStatusExpiration`, + please document the behaviour to the md.

File: coap-gateway/service/config.go [hunk 2]
```diff
@@ -7,6 +7,15 @@ import (
...
+	Enable   bool          `yaml:"enable" json:"enable"`
```
Comment (ondrejtomcik) [inline coap-gateway/service/config.go]: we're using past tense - `Enabled`

File: coap-gateway/service/config.go [hunk 3]
```diff
@@ -7,6 +7,15 @@ import (
...
+	Validity time.Duration `yaml:"validity" json:"validity"`
```
Comment (ondrejtomcik) [inline coap-gateway/service/config.go]: `ExpiresIn`

Comment (ondrejtomcik) [reply coap-gateway/service/config.go]: Is this value somehow related to ping?
e.g. if user configures expiresIn to 5secodns but ping between device and cloud is set to 30 seconds?

File: coap-gateway/service/config.go (hunk 1 above)
Comment (ondrejtomcik) [inline coap-gateway/service/config.go]: There is still pending MR from @Trishia . Is this going to create a conflict? Shouldn't we firstly merge Trishias changes?

File: coap-gateway/service/config.go (hunk 3 above)
Comment (jkralik) [reply coap-gateway/service/config.go]: no it is not related to ping

Comment (ondrejtomcik) [reply coap-gateway/service/config.go]: And shouldn't be? at least, it should be documented how it behaves with combination with ping.

File: coap-gateway/service/config.go (hunk 1 above)
Comment (jkralik) [reply coap-gateway/service/config.go]: Done

File: coap-gateway/service/config.go (hunk 2 above)
Comment (jkralik) [reply coap-gateway/service/config.go]: Done

File: coap-gateway/service/config.go (hunk 1 above)
Comment (jkralik) [reply coap-gateway/service/config.go]: There is lot's of work to finish it - mainly in tests. So I don't want to wait for it. At first @ondrejtomcik pls validate yaml files and than when you are ok with them I can fix tests there.

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**